pages/privacy.tsx

```diff
@@ -1,232 +1,127 @@
-          and affiliates (collectively referred to as "we", "our" or "us") collect, use, share, and protect your
-          information. Please read this notice carefully to understand what we do.
-        </p>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          This Privacy Statement does not apply to Wallora sites, services and products that do not display or link to
-          this statement, or that have their own privacy statements. By using Wallora.com, any Wallora branded product
-          or service, or by providing us your personal information, you agree to this Privacy Statement and agree to be
-          bound by the Wallora Terms of Service. If you do not agree with any term in this Privacy Statement, please do
-          not provide us your personal information or use any Wallora branded site or service.
-        </p>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          Wallora Personal Finance Management Services LLP ('Wallora') owns and operates Wallora.com and other Wallora
-          branded products and services. Note: Our privacy policy may change at any time without prior notification. To
-          make sure that you are aware of any changes, kindly review the policy periodically. This Privacy Policy shall
-          apply uniformly to Wallora desktop website & Wallora mobile applications
-        </p>
+        <div className="font-paragraph">
+          <p className="ml-1 mt-8">
+            For customers of Wallora.com and other Wallora branded products and services (collectively
+            &quot;Wallora&quot;), this Wallora Privacy Statement describes how Wallora Personal Finance Management
+            Services LLP and its subsidiaries and affiliates (collectively referred to as &quot;we&quot;,
+            &quot;our&quot; or &quot;us&quot;) collect, use, share, and protect your information. Please read this
+            notice carefully to understand what we do.
+          </p>
+          <p className="ml-1 mt-8">
+            This Privacy Statement does not apply to Wallora sites, services and products that do not display or link to
+            this statement, or that have their own privacy statements. By using Wallora.com, any Wallora branded product
+            or service, or by providing us your personal information, you agree to this Privacy Statement and agree to
+            be bound by the Wallora Terms of Service. If you do not agree with any term in this Privacy Statement,
+            please do not provide us your personal information or use any Wallora branded site or service.
+          </p>
+          <p className="ml-1 mt-8">
```

Comment:
combine these tailwind classes "ml-1 mt-8" to create a tailwind component "para" as this is repeated throughout the page here